created bulk loading common issues (#4837)
mattermost/docs: source/onboard/bulk-loading-common-issues.rst

.. _bulk-loading-common-issues:

Common Issues

> ---------------

Reviewer: ```suggestion
-------------

```

> Run the bulk import command as the mattermost user. Running it as root or some other user will cause issues with file permissions on imported attachments.

Reviewer: ```suggestion

Run the bulk import command as the *mattermost* user. Running it as *root* or any other user will cause issues with file permissions on imported attachments.

```

Ensure that `file attachments are enabled <https://docs.mattermost.com/configure/configuration-settings.html#allow-file-sharing>`__, that you have enough free space in your `file storage system <https://docs.mattermost.com/configure/configuration-settings.html#file-storage-system>`__ to support the incoming attachments, and that your `maximum file size <https://docs.mattermost.com/configure/configuration-settings.html#maximum-file-size>`__ is appropriate.

> If you are using a version < 5.17, extended `password requirements <https://docs.mattermost.com/configure/configuration-settings.html#password>`__ need to be disabled (no uppercase, no special characters, no numbers, and password length needs to be 5). Any passwords that do not meet the requirements will cause the user import to fail.

Reviewer 2: ```suggestion
If you are using a Mattermost version prior to v5.17, extended `password requirements <https://docs.mattermost.com/configure/configuration-settings.html#password>`__ also need to be disabled (to remove requirements of uppercase, special characters, numbers, and the password length of 5). Any passwords that do not meet the requirements will cause the user import to fail.

```
